src/Keg.sol

```diff
@@ -304,31 +305,30 @@ contract Keg is LibNote {
         for (uint256 i = 0; i < pints.length; i++) {
             Pint memory pint = pints[i];
             uint256 sud;
-            if (i == pints.length - 1) {
-                // Add whatevers left over to the last mug to account for rounding errors
-                sud = sub(rad, suds);
-            } else {
+            if (i != pints.length - 1) {
                 // Otherwise use the share amount
                 sud = mul(rad, pints[i].share) / WAD;
+            } else {
+                // Add whatevers left over to the last mug to account for rounding errors
+                sud = sub(rad, suds);
             }
             suds = add(suds, sud);
-            vat.move(msg.sender, pint.mug, sud);
-            emit PourBeer(pint.mug, sud);
+            vat.move(msg.sender, pint.seat, sud);
+            emit PourBeer(pint.seat, sud);
         }
     }
 
     // Pre-authorize a flight distribution of funds
-    function serve(bytes32 flight, address[] calldata bums, uint256[] calldata shares) external note auth {
-        require(bums.length == shares.length, "Keg/unequal-bums-and-shares");
-        require(bums.length > 0, "Keg/zero-bums");
+    function serve(bytes32 flight, address[] calldata seats, uint256[] calldata shares) external note auth {
+        require(seats.length == shares.length, "Keg/unequal-bums-and-shares");
+        require(seats.length > 0, "Keg/zero-bums");
 
-        // Pints need to add up to 100%
-        // pints[flight] = bums.length;
+        // Pint shares need to add up to 100%
         uint256 total = 0;
-        for (uint256 i = 0; i < bums.length; i++) {
-            require(bums[i] != address(0), "Keg/no-address-0");
+        for (uint256 i = 0; i < seats.length; i++) {
+            require(seats[i] != address(0), "Keg/no-address-0");
             total = add(total, shares[i]);
-            flights[flight].push(Pint(bums[i], shares[i]));
+            flights[flight].push(Pint(seats[i], shares[i]));
         }
```

Comment:
Sorry maybe I should've been more clear, I meant change the function name from `serve` to `seat` and leave `bums` as is.